test fail looks related
```
======================================================================
ERROR: testCreateProgram (custom.openlmis.tests.test_program_product_sync.ProgramSyncTest)
----------------------------------------------------------------------
Traceback (most recent call last):
  File "/home/travis/build/dimagi/commcare-hq/custom/openlmis/tests/test_program_product_sync.py", line 36, in testCreateProgram
    self.assertEqual(product.code, lmis_product.code.lower())
AttributeError: 'NoneType' object has no attribute 'code'
----------------------------------------------------------------------
```

--- a/corehq/apps/products/models.py
+++ b/corehq/apps/products/models.py
@@ -140,7 +140,7 @@ class Product(Document):
         if not code:
             return None
         try:
-            sql_product = SQLProduct.objects.get(domain=domain, code=code)
+            sql_product = SQLProduct.objects.get(domain=domain, code__iexact=code)
         except SQLProduct.DoesNotExist:
             return None
         else:

--- a/custom/openlmis/tests/test_program_product_sync.py
+++ b/custom/openlmis/tests/test_program_product_sync.py
@@ -6,8 +6,8 @@ from custom.openlmis.api import Program
 from corehq.apps.commtrack.tests.util import bootstrap_domain as initial_bootstrap
 from custom.openlmis.commtrack import sync_openlmis_program, bootstrap_domain
 
-
 TEST_DOMAIN = 'openlmis-commtrack-program-test'
+
 
 class ProgramSyncTest(TestCase):
 
@@ -17,7 +17,6 @@ class ProgramSyncTest(TestCase):
         bootstrap_domain(TEST_DOMAIN)
         for product in Product.by_domain(TEST_DOMAIN):
             product.delete()
-
 
     def testCreateProgram(self):
         with open(os.path.join(self.datapath, 'sample_program.json')) as f:

